### PR TITLE
Sd 959 show or hide currency selector as admin

### PR DESCRIPTION
**Pull request by KacperMekarski**

### CODE REVIEW

File: backend/app/controllers/spree/admin/general_settings_controller.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def clear_cache
       end
 
       def render(*args)
-        @preferences_currency |= [:allow_currency_change, :show_currency_selector]
+        @preferences_currency |= [:allow_currency_change]
```
Comment (damianlegawiec) [inline backend/app/controllers/spree/admin/general_settings_controller.rb:30]: @KacperMekarski can we rename it to `show_store_selector` ? To better reflect the current status?

Comment (KacperMekarski) [reply backend/app/controllers/spree/admin/general_settings_controller.rb]: @damianlegawiec done.

File: core/config/locales/en.yml [hunk 2]
```diff
@@ -1570,15 +1567,15 @@ en:
...
-      long_html: Display the currency picker in the main nav bar. This will only display if there are multiple supported currencies, and <strong>Allow currency change</strong> option is enabled.
     show_deleted: Show Deleted
     show_discontinued: Show Discontinued
     show_only_complete_orders: Only show complete orders
     show_only_considered_risky: Only show risky orders
     show_property: Show Property
     show_rate_in_label: Show rate in label
+    show_store_selector:
+      short: Allow Currency Change
+      long: Display the currency picker in the main nav bar and allow users to change their currency via it. In addition, multiple stores must be supported.
```
Comment (damianlegawiec) [inline core/config/locales/en.yml:1578]:
```suggestion
    show_store_selector:
      short: Show Store selector
      long: Display the Store selector in the main nav bar of Storefront and allow users to change Store and Currency
```

**Comment by damianlegawiec** [comment]
@KacperMekarski can we also update this page - https://guides.spreecommerce.org/developer/core/preferences.html#spree-configuration-options ? thanks :)

**Comment by KacperMekarski** [comment]
@damianlegawiec done.

**Comment by KacperMekarski** [comment]
@przemosk
@szymoniwacz
@piotrpanko
@aplegatt 
Anyone? :)

**Review by aplegatt — APPROVED** [review]

**Comment by przemosk** [comment]
code LGTM 👌🏻 , one thing I would like to rise up is preference name, I would like to propose change `show_store_selector` --> `show_store_currency_selector`. I.e  [here](https://github.com/spree/spree/pull/10533/files#diff-428f2740603454a2c74e86c79b3886d9bf0cf20924d51062d1ab6515c9074bacR431) we are saying that preference change Store and Currency but name suggesting only that `Store` will be changed. 
@KacperMekarski please share your thoughts about that :)

**Review by szymoniwacz — APPROVED** [review]
LGTM too! @przemosk's question is worth considering. But I think `show_store_selector` is ok because it is the **store** _with currency_ selector.

**Comment by KacperMekarski** [comment]
@przemosk I will change that name because it's good to mention that changing store has an impact on currency and I feel that in that case currency is even more important.

**Review by przemosk — APPROVED** [review]

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**